src/Controls/tests/DeviceTests/Elements/Label/LabelTests.cs

```diff
@@ -410,6 +411,109 @@ public async Task FontStuffAppliesEvenInHtmlMode()
 			});
 		}
 
+		[Theory (
+#if WINDOWS
+			Skip = "Appears that the layout is not getting rendered on Windows."
```

Comment:
Are you able to try again after Shane's PR reworked the attach and run. I think there may have been an issue with sometimes the layout taking a second so try adding a second delay. I think we have an extension method that waits for a layout pass to run which you could await.

Reply:
Will do!

Reply:
@mattleibow looks like the windows tests pass now :)